cvat-sdk/cvat_sdk/core/client.py

```diff
@@ -43,6 +47,9 @@ class Config:
     verify_ssl: Optional[bool] = None
     """Whether to verify host SSL certificate or not"""
 
+    cache_dir: Path = attrs.field(converter=Path, default=_DEFAULT_CACHE_DIR)
```

Comment:
I'm not sure if the empty value should be allowed now. Probably, it's better to disallow this until the relying code supports it.

Reply:
Empty value? What do you mean?

Reply:
I mean, the user can change the default value and pass the empty string in their code during `Client` configuration. Will this work?

Reply:
`Path('')` is the current directory, so that will set the cache path to the current directory.